### ISSUE TITLE
Install error on Foundry V9

### ISSUE THREAD

**Issue by lewsername00** [free-beer/bh2e#27]
Tried installing BH2E via the System installer in FoundryVTT, but I got this error:

>    The Black Hack 2e version 0.5.2 does not specify a compatibleCoreVersion and may not work with Foundry VTT version 9.280
> 
>     System undefined was installed successfully

Except I don't see anything installed in the Systems menu. I came here and found the user guide, followed the install instructions for The Forge in the [user guide](https://peter-wood-1.gitbook.io/the-black-hack-2e-foundryvtt-system-user-manual/), and now it shows up as Installed in my Forge Bazaar but not in Foundry in the list of available systems.

Posted on Reddit to see what might be the problem, another user provided this insight:

> Looks like they changed the system.json file (which gives Foundry info about the system) to use the new v10 format.
> 
> "compatibility": {
>   "minimum": 9,
>   "verified": "10.286"
> }
> 
> But Foundry v9 doesn't understand that format, and expects something like
> 
> "minimumCoreVersion": 9,
> "compatibleCoreVersion": "10.286"
> 
> So while it says it's compatible with v9, it says so in a way that v9 can't understand.

**Comment by free-beer** [comment]
Yeah, my mistake, it v10 compatible only from release v0.5.0 onward. I'll change the compatible setting in the ``system.json`` file and make another release soon. If you want to install a v9 compatible version use this manifest URL for the install...

https://raw.githubusercontent.com/free-beer/bh2e/v0.4.5/system.json

I think that should work for the stand alone client. To get it working on The Forge I had to remove my existing installation and then install from manifest URL given above. I then stopped and started my world and that seemed to work.
